packages/react-strict-dom/src/native/stylex/index.js

```diff
@@ -354,6 +341,34 @@ function resolveStyle<S: { +[string]: mixed }>(
       continue;
     }
 
+    // Polyfill unitless lineHeight
+    // React Native treats unitless as a 'px' value
+    // Web treats unitless as fontSize multiplier
+    if (propName === 'lineHeight') {
+      if (
+        typeof styleValue === 'number' ||
+        (typeof styleValue === 'string' &&
+          CSSLengthUnitValue.parse(styleValue) == null)
+      ) {
+        const lineHeightValue = parseFloat(styleValue);
+        // Only convert unitless lineHeight if fontSize exists
+        if (style.fontSize != null) {
+          if (style.fontSize instanceof CSSLengthUnitValue) {
```

Comment:
awesome! so now we process the multiplier into a px value based on the current font size?

Reply:
We were doing that before too, but earlier in the translation process. Before, it would be converted to px straight away based on whatever the fontSize was on the element. After, it will not be converted to px until it is being rendered to an element, which should allow the unitless number to be inherited properly.